assignments/callbacks.js

```diff
@@ -19,34 +19,66 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
     console.log(first)
   });
 
-*/
-
 
 function getLength(arr, cb) {
+  return cb(arr.length);
   // getLength passes the length of the array into the callback.
 }
 
 function last(arr, cb) {
+  return cb(arr.length -1)
   // last passes the last item of the array into the callback.
 }
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y + y + y - (2 * y));  
 }
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x*y);
 }
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  let boolo = false;
```

Comment:
to put this one in a simpler term, it just wants to check if an item is present. Keep in mind that in JavaScript, simply passing the items name makes it a true statement while it is there. This could have been written as
`if (list.item) return cb(true)
return cb false`
you don't need the break in your if statement either, that is more so to get you out of switch statements as if they don't find a proper case, they could go into an infinite loop. A regular if statement doesn't have that problem, it just returns false or null if nothing matches.